config/default.yml

```diff
@@ -4700,6 +4700,16 @@ Style/UnlessElse:
   Enabled: true
   VersionAdded: '0.9'
 
+Style/UnlessLogicalOperators:
+  Description: >-
+                 Checks for use of logical operators in an unless condition.
+  Enabled: false
+  VersionAdded: '<<next>>'
+  EnforcedStyle: no_mixed_logical_operator
+  SupportedStyles:
+    - no_mixed_logical_operator
```

Comment:
I'd probably go with names like `forbid_logical_operators` and `forbid_mixed_logical_operators` or something along those lines. Such names seem clearer to me.

Reply:
Sounds good to me. I'll update the PR.